src/database/database.cpp

```diff
@@ -62,8 +62,73 @@ std::string DatabaseManager::retrieve_data(const std::string& table_name) {
     return result;
 }
 
+void DatabaseManager::create_loop(const std::string &loop_name) {
+    std::string sql = "INSERT INTO Loop (loop_name) VALUES (" + loop_name + ");";
+    char* err_msg;
+    int return_status = sqlite3_exec(db_, sql.c_str(), nullptr, nullptr, &err_msg);
+    if (return_status != SQLITE_OK) {
+        std::cerr << "Error retrieving data: " << sqlite3_errmsg(db_) << std::endl;
+        sqlite3_free(err_msg);
+        return;
+    }
+    return;

```

Comment:
We shouldn't have to return anything here as it is a void function

Reply:
I changed this to return an int